Standards/scs-0302-w1-domain-manager-implementation-notes.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Domain Manager implementation notes
 type: Supplement
-track: IaaS
-status: Proposal
+track: IAM
+status: Draft
```

Comment:
```suggestion
status: Stable
```

The related standard [is stable on main](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0302-v1-domain-manager-role.md?plain=1#L4) shouldn't the supplement be too then?

Reply:
No, supplements are meant to be updated more often than standards. That is one reason why we have them. All of them stay in draft state.

Reply:
Oh TIL they don't share the same state as the standard they are supplementing, didn't notice that. My apologies.

> All of them stay in draft state.

Do they? Any supplement I looked at in `main` seems to be in Proposal state though.

Reply:
You are right, and that is strage. I will investigate. My feeling is that a Proposal (as opposed to a Draft) should never enter main branch.